docs/topics/install/docker.rst

```diff
@@ -14,6 +14,7 @@ There are two options for running docker depending on the platform
 you are running.
 
  * Run docker on the host machine directly (recommended)
+    - Update default settings in Docker Desktop - increase RAM limit from 2 GB (preferably 4 GB) in the Resources/Advanced section and click on "Apply and Restart".
```

Comment:
I find this sentence a bit hard to parse. Perhaps it should be "increase RAM limit from 2 GB (we've found that 4 GB works)...".

Reply:
Can I update it to simply "... increase RAM limit from 2 GB to 4 GB..."?